README.md

```diff
@@ -12,125 +13,145 @@ npm install @girder/components
 yarn add @girder/components
 ```
 
-### Use as a prebuilt UMD module
+[VueCLI](https://cli.vuejs.org/) is required for building applications with Girder web components.
+However, a few additional packages must still be manually installed:
+```bash
+npm install -D sass-loader node-sass pug-plain-loader pug
+# or
+yarn add -D sass-loader node-sass pug-plain-loader pug
+```
 
-It's not necessary to install Girder web components yourself, you can import the prebuilt bundle
-into your page by including the following tags:
+### Initialization
 
-```html
-<script src="https://unpkg.com/vue/dist/vue.min.js"></script>
-<script src="https://unpkg.com/@girder/components"></script>
-<link rel="stylesheet" href="https://unpkg.com/@girder/components/dist/girder.css">
-```
+Encapsulating the configuration in another file (typically `src/plugins/girder.js`) is a good practice:
 
-This will expose all the library's components under the global variable `girder`, e.g.
-`girder.components.Upload`.
+```javascript
+import Vue from 'vue';
```

Comment:
```suggestion
// src/plugins/girder.js

import Vue from 'vue';
```

I always really appreciate when documentation with multiple files in-line comment their hypothetical paths at the top.

Reply:
Will fix.